package.json

```diff
@@ -64,5 +64,6 @@
   "lint-staged": {
     "**/*": "prettier --write --ignore-unknown",
     "{src,tests}/**/*.js": "eslint --fix"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
```suggestion
  },
  "packageManager": "^yarn@1.22.19"
```

OK, this is a `corepack` configuration. Keep it, but please add also [`engines`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) configuration.

Reply:
I can't use [Caret Ranges](https://github.com/npm/node-semver#caret-ranges-123-025-004) in `yarn`
it NPM feature.
If I run any `yarn` command like `yarn -v` I get the error 
`Usage Error: Unsupported package manager specification (^yarn@1.22.19,^yarn,1.22.19)`

`engines` - I think it is also a NPM feature. 
`yarn` ignores `engines` and all the node commands work even if I have the wrong version.

so maybe I will revert the `package.json`.

Reply:
OK just keep the `packageManager: "yarn@1.22.19"` as is.